project/AkkaBuild.scala

```diff
@@ -425,9 +425,10 @@ object AkkaBuild extends Build {
     id = "akka-http-marshallers-scala-experimental",
     base = file("akka-http-marshallers-scala"),
     settings = parentSettings ++ Seq(
-      version := streamAndHttpVersion
+      version := streamAndHttpVersion,
+      resolvers += "Typesafe Repo" at "http://repo.typesafe.com/typesafe/releases/"
```

Comment:
Are you sure we need to add the resolver here? I'd have thought we include it already hm..